modules/ROOT/pages/migration/index.adoc

```diff
@@ -44,32 +76,989 @@ a|
 [source, graphql, indent=0]
 ----
 type Movie {
+  title: String!
+}
+----
+a|
+[source, graphql, indent=0]
+----
+type Movie @node {
+  title: String!
+}
+----
+|===
+
+
+=== Removed the deprecated `options` argument
+
+The deprecated `options` argument has been removed.
+
+Consider the following type definitions:
+
+[source, graphql, indent=0]
+----
+type Movie @node {
+  title: String!
+}
+----
+
+The migration is as below:
+
+[cols="1,1"]
+|===
+| Before | After
+
+a|
+[source, graphql, indent=0]
+----
+{
+  movies(options: { first: 10, offset: 10, sort: [{ title: ASC }] }) {
+    title
+  }
+}
+----
+a|
+[source, graphql, indent=0]
+----
+{
+  movies(first: 10, offset: 10, sort: [{ title: ASC }]) {
+    title
+  }
+}
+----
+|===
+
+The `deprecatedOptionsArgument` of `excludeDeprecatedFields` has been removed.
+
+=== Removed the deprecated `directed` argument
+
+The deprecated `directed` argument has been removed from queries. This argument previously allowed you to specify whether relationships should be directed or undirected at query time.
+
+Please use the `queryDirection` argument of the `@relationship` directive instead.
+
+The `directedArgument` setting of `excludeDeprecatedFields` has been removed.
+
+=== Accepted values of the `queryDirection` argument of `@relationship` has been changed
+
+Following the removal of the `directed` argument,
+the `queryDirection` argument of the `@relationship` directive now only accepts two possible values:
+
+- `DIRECTED` (default)
+- `UNDIRECTED`
+
+The following values are no longer supported:
+
+ - `DEFAULT_DIRECTED`
+ - `DEFAULT_UNDIRECTED`
+ - `DIRECTED_ONLY`
+ - `UNDIRECTED_ONLY`
+
+=== The deprecated `typename_IN` filter has been removed
+
+The deprecated `typename_IN` filter has been removed, please use `typename` instead.
+
+[cols="1,1"]
+|===
+| Before | After
+
+a|
+[source, graphql, indent=0]
+----
+{
+  productions(where: { typename_IN: ["Movie", "Series"] }) {
+    title
+  }
+}
+----
+a|
+[source, graphql, indent=0]
+----
+{
+  productions(where: { typename: ["Movie", "Series"] }) {
+    title
+  }
+}
+----
+|===
+
+The `typename_IN` setting of `excludeDeprecatedFields` has been removed.
+
+=== Aggregations are no longer generated for `ID` fields
+
+`ID` fields are excluded from aggregation fields and aggregation filters.
```

Comment:
```suggestion
`ID` fields are excluded from aggregation selection sets and aggregation filters.
```

https://spec.graphql.org/draft/#sec-Language.Fields

Reply:
To be fair, I believe it is more precise fields as terminology as the definition of  `Selection`  contains:
- Field
- FragmentSpread
- InlineFragment
But in our case, we don't generate any of the other two. Doesn't matter anyway!